.evg.yml

```diff
@@ -261,9 +361,9 @@ buildvariants:
     realm_server_url: "http://localhost:8080"
     yarn_url: "https://s3.amazonaws.com/stitch-artifacts/yarn/latest.tar.gz"
     node_version: 12.16.2
-    transpiler_target: node8-linux
-    version: 0.0.0
+    version: 2.0.0-beta.0
```

Comment:
can we leave this at the current version (`1.3.2`) and bump this to `2.0.0-beta.0` as part of a separate ticket for actually releasing? would be nice to keep release commits/changes separate from actual development updates.